pkg/virt-launcher/virtwrap/efi/efi_test.go

```diff
@@ -52,16 +52,18 @@ var _ = Describe("EFI environment detection", func() {
 			efiEnv := DetectEFIEnvironment(arch, ovmfPath)
 			Expect(efiEnv).ToNot(BeNil())
 
-			Expect(efiEnv.Bootable(true)).To(Equal(SBBootable))
-			Expect(efiEnv.Bootable(false)).To(Equal(NoSBBootable))
+			Expect(efiEnv.Bootable(true, false)).To(Equal(SBBootable))
```

Comment:
Indeed with those true/false it was hard to follow the logic. In general no strong objections about the struct. But defined the bool constants simply because of personal preference. Now it looks much better. Thanks.